### ISSUE TITLE
Test cases fail with --path flag in 1.0.0-beta.0

### ISSUE THREAD

**Issue by kandhavivekraj** [kategengler/ember-cli-code-coverage#210]
Test cases fail when I update from 0.4.2 to 1.0.0-beta.0

Command:  `COVERAGE=true node_modules/.bin/ember exam --path='test-dist' --split=4 --parallel --partition=1`

I'm running the above command, and some of my test cases are starting to fail with the error

```
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at writeCoverage (/.../node_modules/ember-cli-code-coverage/lib/attach-middleware.js:29:10)
    at coverageHandler (/.../node_modules/ember-cli-code-coverage/lib/attach-middleware.js:55:3)
    at Layer.handle [as handle_request] (/.../node_modules/express/lib/router/layer.js:95:5)
    at next (/.../node_modules/express/lib/router/route.js:137:13)
    at /.../node_modules/body-parser/lib/read.js:130:5
    at invokeCallback /.../node_modules/raw-body/index.js:224:16)
    at done (/.../raw-body/index.js:213:7)
    at IncomingMessage.onEnd (/.../node_modules/raw-body/index.js:273:7)
    at emitNone (events.js:106:13)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

I guess while passing the `--path` flag, the addons' included hook is not invoked and so `fileLookup`'s value is [null](https://github.com/kategengler/ember-cli-code-coverage/blob/a8eed20591b2eb79c2af598136a51cff32bc7025/index.js#L35) and then in `writeCoverage` method while iterating, `fileLookup` error occurs in [this](https://github.com/kategengler/ember-cli-code-coverage/blob/a8eed20591b2eb79c2af598136a51cff32bc7025/lib/attach-middleware.js#L43) line.

**Comment by kandhavivekraj** [comment]
@rwwagner90 Even in the latest version (`1.0.0-beta.8`), in writeCoverage method fileLookup is being iterated. Hence the same problem [exists](https://github.com/kategengler/ember-cli-code-coverage/blob/2cefa0477d6e14be8b7c812367ceddcc51f45303/lib/attach-middleware.js#L29)

**Comment by rwwagner90** [comment]
@kandhavivekraj I'm not sure if passing `--path` makes it skip the `included` hook or not. I would think it would still execute.

@kategengler @adamjmcgrath do either of you know about this?

**Comment by kandhavivekraj** [comment]
When I place the debugger on the included hook it stops only if `--path` flag is not passed @rwwagner90

**Comment by kategengler** [comment]
As I understand it, the `--path` option is to reuse an existing build, in which case it is fundamentally incompatible with `ember-cli-code-coverage`. In order to generate coverage this addon needs to add instrumentation during the build.

**Comment by kandhavivekraj** [comment]
The build was generated with COVERAGE=true as specified in the README.

Also, coverage works till version 0.4.2.

**Comment by ming-codes** [comment]
I had a previous issue on this #181

I don't think the fix is proper.

**Comment by ming-codes** [comment]
What is everyone's Ember CLI version?

**Comment by lan-nguyen91** [comment]
@ming-codes this is ugly, but we have to fork and do this

![image](https://user-images.githubusercontent.com/14827876/65806896-c7ef0e00-e140-11e9-92d5-92255982b09e.png)

**Comment by abishek-srinivasan** [comment]
I am using --path option in my test command (COVERAGE=true ember exam --path='./testOutput' --split=20 --parallel=1). I faced this issue when I updated the code coverage version from 0.4.2 to 1.0.0-beta.0. But now, I am using the latest version "ember-cli-code-coverage": "1.0.0-beta.8" and I still face the exact error mentioned in https://github.com/kategengler/ember-cli-code-coverage/issues/210#issue-395907284
As @kandhavivekraj mentioned in https://github.com/kategengler/ember-cli-code-coverage/issues/210#issuecomment-451449246, the problem still persist.

Is this issue fixed or forking the repo is the only possible solution here?
@kategengler @rwjblue  @rwwagner90 and @ming-codes
